Flutter_TDD_Architecture_Course_notes.md

```diff
@@ -277,3 +277,302 @@ class GetRandomNumberTrivia extends UseCase<NumberTrivia, NoParams> {
 And now the **second** test will pass.
 
 ![Second_test_pass](https://user-images.githubusercontent.com/27420533/74590474-aa92e180-5006-11ea-8a88-ef8a9a6311e7.png)
+
+#### 4. Data Layer Overview & Models
+> Video 4: https://www.youtube.com/watch?v=keaTZ9M_U1A&t=1516s
+
+In the `fourth` tutorial start by giving a brief explanation of how the models work.
+
+The models are like entities but with more features in the case of this specific application will deal with making the data conversation in Json to Dart.
+
+After this explanation is necessary to create a file inside the **models** folder with the name `number_trivia_model.dart`.
```

Comment:
The words "After this explanation" do not add to the remainder of the sentence. 
You can convey the same meaning/message with fewer words:
```suggestion
Create a file in the **models** folder named `number_trivia_model.dart`.
```